### PR TITLE
Fixes #479 to  add document partitioned DBs + Mango content

### LINKED ISSUE #479: Document partitioned DBs + Mango
Left out of #385

### PR DESCRIPTION
**Pull request by lbboe**
## Overview
 Per issue #479, update the document partitioned DBs + Mango content. This content is not currently documented. 

## GitHub issue number

Issue #479 

## Checklist

- [ ] Documentation is written and is accurate;
- [ ] `make check` passes with no errors

### CODE REVIEW

File: src/api/database/find.rst [hunk 1]
```diff
@@ -943,6 +943,12 @@ built using MapReduce Views.
     :query json partial_filter_selector: A :ref:`selector <find/selectors>`
         to apply to documents at indexing time, creating a
         :ref:`partial index <find/partial_indexes>`. *Optional*
+    :query boolean partitioned: Determines whether a JSON index is partitioned
+        or global. To create a partitioned index, set the ``"partitioned"`` field
+        to ``true``. To create a global index on a partitioned database, specify 
+        ``false`` for the ``"partitioned"`` field. If you specify ``true``
+        for the  ``"partitioned"`` field on an unpartitioned database, an
+        error occurs.
```
Comment (flimzy) [inline src/api/database/find.rst]: Is there a default value, or is this value required in all cases?

File: src/api/partitioned-dbs.rst [hunk 2]
```diff
@@ -194,3 +194,38 @@ See the guide for
           ],
           "total_rows": 4
         }
+.. _api/partitioned/find:
+
+``/db/_partition/partition_id/_find``
+=====================================
+
+.. http:get:: /{db}/_partition/{partition_id}/_find
+    :synopsis: Find a partition query by ID
```
Comment (flimzy) [inline src/api/partitioned-dbs.rst]: This description seems a bit unclear to me.  Is this finding a _partition query_ (if so, what is that?), or is it executing a find query only on a specific partition?

**Comment by wohali** [comment]
@lbboe Thank you for a fast turnaround on this!

Looks like there is a newline error that is preventing this from landing:

```
[2020-01-31T21:29:26.586Z] /home/jenkins/workspace/cm1_CouchDB-Documentation_PR-491@2/src/api/partitioned-dbs.rst
[2020-01-31T21:29:26.586Z]   line 232 : last line should ends with newline character
```

If you can add the newline to the end of this file, we can get this merged straight away.

Thanks again!

**Comment by lbboe** [comment]
Here you go, @wohali. Thanks again!

**Pull request merged**